test/metabase/test/util.clj

```diff
@@ -100,72 +94,94 @@
 
 (defn- rasta-id [] (user-id :rasta))
 
+(defn- default-updated-at-timestamped
```

Comment:
the `:now` function in progress will return the now at the beginning of the transaction, so for tests that really care about the order of created_at it will not return the order as it's created in real time.

so for testing purposes, we get the time from clojure land to preserve the order of things